conformance/generated/google/protobuf/any.luau

```diff
@@ -13,6 +13,19 @@ type _AnyImpl = {
 	jsonEncode: (self: Any) -> { [string]: any },
 	jsonDecode: (input: { [string]: any }) -> Any,
 	descriptor: proto.Descriptor,
+
+	-- Pack a message into an Any.
+	--
+	-- If typePrefix is not provided, defaults to "apis.roblox.com/cloud".
```

Comment:
I thought we didn't want to go with this?

Reply:
Oh right, I left this in as a placeholder and forgot to remove it - removed.  I'll probably send a follow-up PR that makes the type URL optional if the plugin gets a default as an argument.